### PR TITLE
Make compact-gate tests work for n=100

### PR DESCRIPTION
**Pull request by richajaindce**
Fix for #921

### CODE REVIEW

File: ipa-core/tests/common/mod.rs [hunk 1]
```diff
@@ -203,7 +203,7 @@ pub fn test_ipa(mode: IpaSecurityModel, https: bool) {
 }
 
 pub fn test_ipa_with_config(mode: IpaSecurityModel, https: bool, config: IpaQueryConfig) {
-    const INPUT_SIZE: usize = 10;
+    const INPUT_SIZE: usize = 100;
```
Comment (richajaindce) [inline ipa-core/tests/common/mod.rs:206]: @akoshelev not sure if you wanted to change this SIZE to 100 but 100 shouldn't be too big a number to cause slowdown. Happy to revert if needed

File: scripts/collect_steps.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "--num-multi-bits",
     "3",
 ]
-QUERY_SIZE = 10
+QUERY_SIZE = 100
```
Comment (richajaindce) [inline scripts/collect_steps.py:22]: Changed this here to avoid issues in future

Comment (akoshelev) [reply scripts/collect_steps.py:22]: would it be sufficient to run malicious IPA for 1M rows? It seems that number of steps synthesized depends on the depth of the circuit - in this case would we be better generating all of them, similarly to what we did before?

**Comment by richajaindce** [comment]
#927 is a better fix

**Pull request closed without merging**